blog/_posts/2021-05-18-argocd-private-clusters.md

```diff
@@ -0,0 +1,414 @@
+---
+layout: post
+title: Bring GitOps to your private Kubernetes clusters with ArgoCD and inlets PRO
+description: Learn how to apply GitOps on multiple private Kubernetes cluster with a single Argo CD installation and inlets PRO
+author: Johan Siebens
+tags: inlets-pro hybrid-cloud multi-cloud gitops argocd
+author_img: jsiebens
+image: /images/2021-05-18-argocd-private-clusters/background.jpg
```

Comment:
This image is too large - at nearly 1MB it will affect loading times and page ranking.